core_lib/interface/scribblearea.cpp

```diff
@@ -617,6 +621,9 @@ void ScribbleArea::mouseMoveEvent( QMouseEvent *event )
 
     currentTool()->mouseMoveEvent( event );
 
+    // used to clear and update canvas cursor
+    update();
```

Comment:
Is it possible to update only a small rectangle rather than update the whole canvas?